### DISCUSSION TITLE
Help with organisation transfer/reclaim for `gocd`

### DISCUSSION THREAD

**Discussion by chadlwilson** [artifacthub/hub#3930] [General]
Hello - can someone please help me figure out how to "reclaim" the organisation https://artifacthub.io/packages/search?org=gocd&sort=relevance&page=1 so that I can claim ownership of https://artifacthub.io/packages/helm/gocd/gocd and https://github.com/gocd/helm-chart ?

I have tried to contact the likely earlier maintainers, but we cannot figure out which GitHub or other accounts may have been linked to this Artifact Hub organisation such that I can be added by them (as current primary maintainer of the chart) and thus do all the `artifacthub-repo.yml` magic ~([need the repository ID at the very least](https://github.com/gocd/helm-chart/blob/gh-pages/artifacthub-repo.yml)).~

**Comment by tegioz** [comment]
Hi @chadlwilson 👋

Please note that you don't need the ID of a repository to claim its ownership. As long as the email of the user doing the request matches one of the `owners` defined in the `artifacthub-repo.yml` file, the process should complete successfully 🙂 (you can find more details about the process [here](https://artifacthub.io/docs/topics/repositories/#ownership-claim)).

Unfortunately it's not possible to claim the ownership of an organization in an automated way like we do for repositories. But I could try to put you in contact with the current members of the organization if that helps.

**Reply by chadlwilson** [reply]
OK, thanks - after re-reading I note that's been clarified that you don't necessarily need the ID.

However, I think the problem is that it may already have been claimed by the other org, and I am not a member of that org. I don't really want to transfer it to myself :-)

My end goal is to get the Verified Publisher and Official statuses sorted - I believe the next step is to get into that org somehow!

If you could help share the current organisation members, or show me how to find them, that would indeed help.

**Reply by tegioz** [reply]
Hi @GaneshSPatil @marques-work @kritika-singh3 @maheshp 👋

It'd be great if you could help us a bit with this 😇

Thanks!

**Reply by chadlwilson** [reply]
Thanks! I've already talked to @GaneshSPatil about this separately, and he didn't seem to have access any longer.

If the ArtifactHub accounts are linked to email address via a federated login, in some cases folks may have changed email address for their GH account (or lost access to that GH account) and I wonder if that means losing the ArtifactHub link to the account.

Now I know who is theoretically there I can also check in with the others offline.

**Reply by chadlwilson** [reply]
I am a member of the org now, and all is good with the world. Thanks for the assist!

**Reply by tegioz** [reply]
Awesome, no worries!
